dnf-behave-tests/dnf/upgrade-from-noarch.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Upgrade package from noarch to arch
+
+Background: Install package to upgrade
+   Given I use repository "upgrade-from-noarch"
+    When I execute dnf with args "install it-1.0"
+    Then the exit code is 0
+     And DNF Transaction is following
+         | Action        | Package         |
+         | install       | it-1.0-1.x86_64 |
+
+Scenario: Upgrade to rpm from file which is not noarch
+   Given I execute dnf with args "install it.noarch --downloadonly --downloaddir={context.dnf.installroot}/it"
```

Comment:
"Given I **successfully** execute..."?
Or better, you do not need to download the package, you can directly use fixtures path in the following `I execute dnf with args` step: `{context.dnf.fixturesdir}/repos/upgrade-from-noarch/noarch/it-2.0-1.noarch.rpm`

Reply:
thanks, I fixed it